src/styles/card.css

```diff
@@ -0,0 +1,100 @@
+.card {
+  box-sizing: border-box;
```

Comment:
It is a good practice to add `box-sizing: border-box;` on all elements `* {...}`